### PR TITLE
Add Doc for loading Snippets in Subfolder

### PR DESCRIPTION
**Pull request by TheCadien**
| Q | A
| --- | ---
| Related PRs | sulu/sulu-docs#613
| License | MIT

#### What's in this PR?

As discussed in the PR https://github.com/sulu/sulu-docs/pull/613. 
The Seperate Configuration for loading Snippets from a Subfolder

### CODE REVIEW

File: cookbook/override-dir-structure.rst [hunk 1]
```diff
@@ -36,5 +36,23 @@ generated it with ``npm install`` and ``npm run build`` into your new directory.
                 sulu_admin:
                     json_manifest_path: "%kernel.project_dir%/public/your/new/path/manifest.json"
 
+
+Load Snippets from a Subfolder
```
Comment (niklasnatter) [inline cookbook/override-dir-structure.rst]: Thanks for your PR!

I would prefer creating a `SnippetBundle` documentation (similar to the `LocationBundle` documentation, see also https://github.com/sulu/sulu-docs/pull/613#discussion_r547400063) and add this section there. I think that is the better approach if we (you 😉 ) want to extend the documentation in the future 🙂

**Comment by TheCadien** [comment]
After Merging of https://github.com/sulu/sulu-docs/pull/613 i change the Doc in that Bundle Doc too.

**Comment by TheCadien** [comment]
@nnatter  After Merging of #613 i change the Doc in that Bundle Doc too.

File: cookbook/override-dir-structure.rst [hunk 2]
```diff
@@ -35,6 +35,5 @@ generated it with ``npm install`` and ``npm run build`` into your new directory.
             packages:
                 sulu_admin:
                     json_manifest_path: "%kernel.project_dir%/public/your/new/path/manifest.json"
```
Comment (niklasnatter) [inline cookbook/override-dir-structure.rst]:
```suggestion
                    json_manifest_path: "%kernel.project_dir%/public/your/new/path/manifest.json"

```

File: bundles/snippet.rst [hunk 3]
```diff
@@ -74,3 +74,22 @@ Properties
 ----------
 
 Properties are the same as Page :doc:`../book/templates`.
+
+
+
+Load Snippets from a Subfolder
```
Comment (niklasnatter) [inline bundles/snippet.rst]:
```suggestion
Properties are the same as Page :doc:`../book/templates`.

Load Snippets from a Subfolder
```

**Comment by niklasnatter** [comment]
Thanks! 🎉

**Pull request merged**